Sources/TuistKit/Utils/ManifestGraphLoader.swift

```diff
@@ -143,8 +143,10 @@ public final class ManifestGraphLoader: ManifestGraphLoading {
         )
 
         // Lint Manifests
-        let lintingIssues = manifestProjects.flatMap { manifestLinter.lint(project: $0.value) }
-        try lintingIssues.printAndThrowErrorsIfNeeded()
+        let workspaceLintingIssues = manifestLinter.lint(workspace: allManifests.workspace)
+        let projectLintingIssues = manifestProjects.flatMap { manifestLinter.lint(project: $0.value) }
+        try projectLintingIssues.printAndThrowErrorsIfNeeded()
+        let lintingIssues = workspaceLintingIssues + projectLintingIssues
```

Comment:
The `printAndThrowErrorsIfNeeded()` method stops the generation process by throwing in the event lint issues with severity `error` are encountered. 

We can print and throw the combined linting issue as the workspace linter may also produce error lint issues.

```suggestion
        let lintingIssues = workspaceLintingIssues + projectLintingIssues
        try lintingIssues.printAndThrowErrorsIfNeeded()

```